src/components/therapistProfile/therapistHeader/index.tsx

```diff
@@ -0,0 +1,173 @@
+import {
+  Container, Box, Typography, Skeleton, Button,
+} from '@mui/material';
+import React, {
+  useState, useEffect, Dispatch, SetStateAction,
+} from 'react';
+import { useParams } from 'react-router-dom';
+import { TextFieldEdited, SessionReservationModal, AppointmentsModal } from '../..';
+import {
+  TherapistData, BoxStyle, ButtonStyle, TypographyStyle,
+} from './classes';
+import { axiosInstance } from '../../../utils/apis';
+import BioEditor from '../changeBio';
+import ChangePhoto from '../changePhoto';
+
+interface Props {
+  isEditable: boolean,
+    setError: Dispatch<SetStateAction<boolean>>,
+}
+
+const TherapistHeader: React.FC<Props> = ({ isEditable, setError }) => {
+  const { id } = useParams();
+  const [dataFromTherapist, setDataFromTherapist] = useState<TherapistData | null>(null);
+  const [name, setName] = useState<string>('');
+  const [major, setMajor] = useState<string>('');
+  const [hourlyRate, setHourlyRate] = useState<number>(0);
+  const [textBio, setTextBio] = useState<string>('');
+  const [image, setImage] = useState<string>('');
+  const [openModal, setOpenModal] = useState(false);

```

Comment:
please use descriptive names fro your state's,
```suggestion
  const [showReservationModal, setShowReservationModal] = useState(false);

```
same for 
```js
  const [open, setOpen] = useState(false);
```